src/App.js

```diff
@@ -1,13 +1,154 @@
-import React from 'react';
+import React from "react";
+import CreateList from "./comps/List/CreateList.js";
+import CreateForm from "./comps/Form/CreateForm.js";
+import "./App.css";
+
+const defaultData = JSON.stringify([
+  { name: "Navigate to my To Do list", complete: true },
+  {
+    name: "Learn I can click an item to complete it",
+    complete: false
+  },
+  {
+    name: 'Click "Clear Completed" to remove completed items.',
+    complete: false
+  },
+  {
+    name: "Highlight items then clear to remove multiples at once",
+    complete: false
+  }
+]);
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      todo: [{ name: "Loading list....", complete: false }],
+      typing: "",
+      search: ""
+    };
+
+    this.saveStateToStorage = this.saveStateToStorage.bind(this);
+  } // constructor
+
+  componentDidMount() {
```

Comment:
too many things to address them all but looking over this entire page I can tell your skills with React and handling Class components is solid. I have nothing to really critique or mention. Well done